### ISSUE TITLE
Hello, What did I miss，Why didn't I  get the same results as  v3_rain100H.pth? 

### ISSUE THREAD

**Issue by Jackyinuo** [tsingqguo/efficientderain#16]
I'm using ”train.sh” just like the file I just downloaded. 
Can you provide some information on how to train the same test results with thedatasets you specify？？

**Comment by MKFxxL** [comment]
Thank you for your question
1. Please make sure the Rain100H dataset you use is old_version
2. For v3-rain100H, we trained 3500 epochs without ssimloss and then fine-tune 2500 epochs with ssimloss to get the final result. But I think you can get a smiliar result by directly training v3 for 2000~5000 epochs with learning rate 0.0002

**Comment by Jackyinuo** [comment]
Thank you so much for your guidance. I thought 250&nbsp;epochs&nbsp;&nbsp;of training would get&nbsp; 30+ and 0.89+ testing in Rain100h. This time, I'll try it again with the following Settings.
'''python ./train2.py \ --epochs 5000 \ --save_mode 'epoch' \--save_by_epoch 20 \--save_by_iter 10000 \--lr_g 0.0002 \--b1 0.5 \--lr_decrease_epoch 50 \ --rainaug "false" \


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "tsingqguo/efficientderain"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2021年2月22日(星期一) 晚上8:16
收件人:&nbsp;"tsingqguo/efficientderain"<efficientderain@noreply.github.com&gt;;
抄送:&nbsp;"水"<2448174049@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [tsingqguo/efficientderain] Hello, What did I miss，Why didn't I  get the same results as  v3_rain100H.pth?  (#16)





 
Thank you for your question
  
Please make sure the Rain100H dataset you use is old_version
 
For v3-rain100H, we trained 3500 epochs without ssimloss and then fine-tune 2500 epochs with ssimloss to get the final result. But I think you can get a smiliar result by directly training v3 for 2000~5000 epochs with learning rate 0.0002

**Comment by SuferDude** [comment]
能不能估计一下efficientderain算法在2080Ti上的速度大概是多少？
